templates/command/new/headersOnly/getChannel.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import sys
+
+preReleaseTagFile = open(".version/PreReleaseTag.txt", "r")
+preReleaseTag = preReleaseTagFile.read()
```

Comment:
jedną linią jakby to opędzlować to wydaje mi się, że też całkiem czytelne by to było